src/material/chips/chip-input.ts

```diff
@@ -7,20 +7,32 @@
  */
 
 import {BooleanInput, coerceBooleanProperty} from '@angular/cdk/coercion';
-import {hasModifierKey, TAB} from '@angular/cdk/keycodes';
-import {Directive, ElementRef, EventEmitter, Inject, Input, OnChanges, Output} from '@angular/core';
+import {BACKSPACE, hasModifierKey, TAB} from '@angular/cdk/keycodes';
+import {
+  AfterContentInit, 
+  Directive,
+  ElementRef,
+  EventEmitter,
+  Inject,
+  Input,
+  OnChanges,
+  OnDestroy,
+  Output
+} from '@angular/core';
 import {MatChipsDefaultOptions, MAT_CHIPS_DEFAULT_OPTIONS} from './chip-default-options';
 import {MatChipList} from './chip-list';
 import {MatChipTextControl} from './chip-text-control';
 
-
 /** Represents an input event on a `matChipInput`. */
 export interface MatChipInputEvent {
   /** The native `<input>` element that the event is being fired for. */
   input: HTMLInputElement;
 
   /** The value of the input. */
   value: string;
+
+  /** Call to clear the value of the input */
+  clearInput(): void;
```

Comment:
rather than having the `clearInput` method on the event, how about adding a reference to the `MatChipInput` and putting the method on the directive instead. Then people would do something like this: `event.chipInput.clear()`

Reply:
Good idea! will change it 👍🏻

Reply:
What about the input itself? the `HTMLInputElement`... should it still be part of the event?

Reply:
We can't remove it without going through a deprecation period. You can mark it as deprecated:
```ts
/**
 * @deprecated use `chipInput.inputElement`
 * @breaking-change 13.0.0 remove this property.
 */
```

(You'll need to change `_inputElement` on `MatChipInput` to public and remove the underscore)